### PR TITLE
Changes to develop

### PR DESCRIPTION
**Pull request by 0xTimepunk**
## Description

......

## Changes

I included a new ABI for the NFC Registry Contract which contains a change to one of the environmental variables needed to run the adapter. The adapter won't take a NFC address anymore but  just the registry contract address

## Steps to Test

 yarn test $adapter/test/integration

## Quality Assurance

- [x] If a new adapter was made, or an existing one was modified so that its environment variables have changed, update the relevant `<ADAPTER_PACKAGE>/schemas/env.json` and `<ADAPTER_PACKAGE>/README.md`
- [x] If a new adapter was made, or an existing one was modified so that its environment variables have changed, update the relevant `infra-k8s` configuration file.
- [x] The branch naming follows git flow (`feature/x`, `chore/x`, `release/x`, `hotfix/x`, `fix/x`) or is created from Clubhouse/Shortcut
- [x] This is related to a maximum of one Clubhouse/Shortcut story or GitHub issue
- [x] Types are safe (avoid TypeScript/TSLint features like any and disable, instead use more specific types)

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -0,0 +1,10778 @@
+{
```
Comment (justinkaseman) [inline package-lock.json:1]: Hey @0xTimepunk, thanks for the PR!

This file isn't needed. We use `yarn` on the monorepo and the lock file sits at the root.

**Comment by boxhock** [comment]
Hey @0xTimepunk, it seems this PR got lost in the mix. Are the changes still relevant? Do you want to update this PR and we'll give it a proper review?

**Comment by austinborn** [comment]
Closing this PR, but we're happy to re-open it if you can update with the latest from our `develop` branch.

**Comment by 0xTimepunk** [comment]
Hi @boxhock and @austinborn 

We took some architecture changes in spectral that will be reflected probably in an updated version of this adapter. We will update the latest from develop and resume work once we are ready.

Thanks

**Comment by austinborn** [comment]
> Hi @boxhock and @austinborn
> 
> We took some architecture changes in spectral that will be reflected probably in an updated version of this adapter. We will update the latest from develop and resume work once we are ready.
> 
> Thanks

Sounds great! I'll reopen this for now so we don't lose it.

**Comment by 0xTimepunk** [comment]
I have merged upstream changes and we will conduct some extra upgrades to the adapter in preparation to our open-beta launch

**Comment by boxhock** [comment]
Hey @0xTimepunk it looks like this PR has gone stale for some time. Are we good to close this for now?

**Comment by boxhock** [comment]
Closing this PR as it's been stale for 3 months. Definitely open to re-open this PR once it's ready to be updated!

**Pull request closed without merging**